Classes/Fluid/ViewHelper/ComponentRenderer.php

```diff
@@ -133,6 +134,9 @@ public function render()
             $renderingContext->setControllerContext($this->renderingContext->getControllerContext());
         }
         $renderingContext->setViewHelperVariableContainer($this->renderingContext->getViewHelperVariableContainer());
+        if (static::shouldUseTemplatePaths()) {
+            $renderingContext->setTemplatePaths($this->renderingContext->getTemplatePaths());
```

Comment:
Could you check if this would work as well?
```php
$renderingContext->getTemplatePaths()->setPartialRootPaths(
    $this->renderingContext->getTemplatePaths()->getPartialRootPaths()
);
```